.pre-commit-config.yaml

```
@@ -0,0 +1,56 @@
+# See https://pre-commit.com for more information
+# See https://pre-commit.com/hooks.html for more hooks
+repos:
+- repo: https://github.com/pre-commit/pre-commit-hooks
+  rev: v4.1.0
```

```suggestion
  rev: v4.3.0
```
Update to the most recent version (fixes an error that can and did crop up in testing)